docs/sources/flow/reference/components/otelcol.receiver.vcenter.md

```diff
@@ -92,6 +96,77 @@ isn't provided, TLS won't be used for connections to the server.
 
 {{< docs/shared lookup="flow/reference/components/otelcol-tls-config-block.md" source="agent" version="<AGENT VERSION>" >}}
 
+### metrics block
+
+Name | Type | Description | Default | Required
```

Comment:
Do you mind making `metric` in the `Type` column a hyperlink to `#### metric block` please?
BTW After we get consensus on #5764, I might need to rework this doc to be in line with that future desicion.